js/pokedex-moves.js

```diff
@@ -9,8 +9,34 @@ var PokedexMovePanel = PokedexResultPanel.extend({
 		buf += '<a href="/" class="pfx-backbutton" data-target="back"><i class="fa fa-chevron-left"></i> Pok&eacute;dex</a>';
 		buf += '<h1><a href="/moves/'+id+'" data-target="push" class="subtle">'+move.name+'</a></h1>';
 
-		if (move.id === 'magikarpsrevenge') buf += '<div class="warning"><strong>Note:</strong> Made for testing on Pok&eacute;mon Showdown, not a real move.</div>';
-		else if (move.isNonstandard) buf += '<div class="warning"><strong>Note:</strong> This is a made-up move by <a href="http://www.smogon.com/cap/" target="_blank">Smogon CAP</a>.</div>';
+		if (move.id === 'magikarpsrevenge') {
+			buf += '<div class="warning"><strong>Note:</strong> Made for testing on Pok&eacute;mon Showdown, not a real move.</div>';
+		} else if (move.isNonstandard) {
+			buf += '<div class="warning"><strong>Note:</strong> ';
+			switch (move.isNonstandard) {
+			case 'Past':
+				buf += 'This move is only available in past generations.';
+				break;
+			case 'Future':
+				buf += 'This move is only available in future generations.';
+				break;
+			case 'Unobtainable':
+				buf += 'This move is not obtainable';
+				if (move.isMax) buf += ' by any means because it does not have an index number';
```

Comment:
```suggestion
				if (move.isMax) {
					buf += 'This move can't be learned normally, it can only be used by ' + (move.isMax === true ? 'Dynamaxing' : 'Gigantamaxing') + '.';
				} else if (move.isZ) {
					buf += 'This move can't be learned normally, it can only be used with a Z-crystal.';
				} else {
					buf += 'This move can't be learned normally.';
				}
```

Reply:
Z-Moves aren’t marked as unobtainable

Reply:
They should be, then. Do this anyway, we can mark Z-moves as unobtainable later.